third_party/move/move-core/types/src/vm_status.rs

```diff
@@ -571,11 +571,21 @@ pub enum StatusCode {
     MULTISIG_TRANSACTION_INSUFFICIENT_APPROVALS = 34,
     MULTISIG_TRANSACTION_PAYLOAD_DOES_NOT_MATCH_HASH = 35,
     GAS_PAYER_ACCOUNT_MISSING = 36,
+    // ZkID account specific errors
```

Comment:
cc @georgemitenkov, is it kosher what we are doing here? We are adding some new error codes for zkID signature verification

Reply:
Do we need all of these? Like some things e.g. `FAILED_TO_GET_ON_CHAIN_CURRENT_TIME` or `FAILED_TO_GET_ON_CHAIN_JWKS` should never happen because those are configs, right? There is also a substatus in `VMStatus::Error` which can be used to have more context, for example? Also @runtian-zhou @wrwg @vgao1996 can have more input.

Reply:
reverted vm_status